pkg/controller/resourcequota/resource_quota_controller_patch.go

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
this is the script output? Would call it `<drop>`.

Reply:
Will do